src/bot.js

```diff
@@ -16,6 +16,9 @@ function randomInteger(min, max) {
 function getRandomLocation() {
   return LOCATIONS[randomInteger(0, LOCATIONS.length - 1)];
 }
+function getRandomRebirhtLocation() {
```

Comment:
```suggestion
function getRandomRebirthLocation() {
```

If you add this function that returns rebirth locations it doesn't really make sense for the other function to be called `getRandomLocation` anymore...?

Maybe what you could do is have `getRandomLocation` take in an argument which would be the `map` and then you could define `getRandomVerdanskLocation()` and `getRandomRebirthLocation()`